src/SourceBuild/content/repo-projects/dotnet.proj

```diff
@@ -12,7 +12,6 @@
 
   <ItemGroup>
     <RepositoryReference Include="sdk" />
-    <RepositoryReference Include="scenario-tests" />
```

Comment:
is removing this intentional?

Reply:
Yes, scenario-tests now gets built in the tests step similar to the other test projects. This was necessary as it used a different cache during build as during testing. I wanted to make this change anyway to better split the composition of the product from the testing process.